homeassistant/components/zeversolar/strings.json

```diff
@@ -0,0 +1,20 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "data": {
+          "ip_address": "[%key:common::config_flow::data::ip%]"
```

Comment:
A hostname doesn't work? As in, in that case, we could just ask for "Host", which sounds a bit friendlier IMHO (and implies both IP & hostname could work).